server/chat-plugins/calculator.ts

```diff
@@ -132,17 +139,26 @@ export const commands: ChatCommands = {
 	math: "calculate",
 	calculate(target, room, user) {
 		if (!target) return this.parse('/help calculate');
+		const targets = target.split(',');
```

Comment:
idk how to feel about using comma as a separator.

Reply:
I guess it risks being misleading for people who are used to , are the decimal separator. What other options are there, though? Using | ? Putting the base in the command name, like math16?

Reply:
I would probably want to do it Google-like, where you'd postfix "to hex", "to binary", "to octal".

Also I would automatically show both bases.

So it'd display:

`0x8 + 0x8 = 0x10 = 16`